contracts/ubiquistick/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "clean": "hardhat clean",
-    "build": "TS_NODE_TRANSPILE_ONLY=1 hardhat compile",
+    "build": "TS_NODE_TRANSPILE_ONLY=1 yarn hardhat compile",
```

Comment:
I was experimenting with this, not sure if I'm to blame for this line, but curious to know if this is required or not?